dev/test-zeekoe.py

```diff
@@ -127,16 +130,17 @@ def run_command(cmd, verbose):
     while True:
         try:
             output = process.stdout.readline()
-            if process.poll() is not None:
-                break
             if output:
                 output_text = output.strip().decode('utf-8')
                 log("-> %s" % output_text, verbose)
+            error = process.stderr.readline()
+            if error:
+                err("-> %s" % error.strip().decode('utf-8'))
```

Comment:
What is the benefit from logging the error in this way, as far as I can see, the test won't fail because of it? I suppose it's just clarity, and not necessarily to improve the tests?

Reply:
No, this does not cause the test to fail, but it does log the error as it happens instead of after the process ends completely. Before, it was hard to tell even with manual inspection whether an error had happened (especially if there was more than one). I think this was written with the incorrect assumption that logs to `stderr` are always associated a fatal error, or something; in fact, they sometimes happen but the program continues.